stdlib/templates/hashtbl.template.mli

```diff
@@ -186,7 +186,7 @@ val randomize : unit -> unit
     It is recommended that applications or Web frameworks that need to
     protect themselves against the denial-of-service attack described
     in {!create} call [Hashtbl.randomize()] at initialization
-    time.
+    time before any children domains are created.
```

Comment:
I believe "child domains" is more widely used than "children domains", "child" being an adjective here.  An example from an Oracle documentation: "When a UNIX process initiates one or more dependent processes, we call these child processes, or children."

This said you could also write "before any domains are created".

Reply:
I believe that `child domains` is better than `any domains` since there is no ambiguity on the first domain. And yes `child domains` sound better than `children domains`.

Reply:
But doesn't `child` introduce a concept that doesn't really exist ? AFAIK there no notion of parent/child relationship in domains.

Yet another proposal: `before any new domain is created.`

Reply:
There is a notion of parent and child domains introduced by domain local storage. Nevertheless, you are right that it better to not introduce adjacent concepts in the description. At the same time, speaking of `before new domains` muddles a bit the notion of `before`. So maybe `before any domains` is the right compromise since the main domain cannot be created by users.